deploy_cluster.yml

```diff
@@ -71,12 +58,23 @@
     - debug: False
     - boot_iso_url: "{{ discovery_iso_server }}/{{ discovery_iso_name }}"
 
-- name: Installing the cluster
+- name: pause to let bmc reboot
```

Comment:
Pauses are unreliable, and at the bare minimum should be configurable,
You generally want to just check if its come up in a way which is not destructive.